pkg/server/container_create_windows.go

```diff
@@ -527,39 +527,70 @@ func (c *criService) generateContainerSpec(id string, sandboxID string, sandboxP
 		g.SetProcessUsername(username)
 	}
 
-	// For both LCOW and WCOW, devices are passed through from the host to the
-	// container via the OCI Windows.Devices field
-	if err := addOCIWindowsDevices(&g, config.Devices); err != nil {
+	if err := addOCIDevices(&g, plat.OS, config.Devices); err != nil {
 		return nil, err
 	}
 
 	return g.Config, nil
 }
 
-// getWindowsDeviceInfo is a helper function that returns a spec specified device's
+// getCRIDeviceInfo is a helper function that returns a spec specified device's
 // prefix and device identifier. A prefix is any string before "://" in the spec
 // device's `HostPath`.
-func getWindowsDeviceInfo(hostPath string) (string, string, error) {
+func getCRIDeviceInfo(hostPath string) (string, string, error) {
 	substrings := strings.SplitN(hostPath, "://", 2)
 	if len(substrings) <= 1 {
 		return "", "", fmt.Errorf("failed to parse device information for %s", hostPath)
 	}
 	return substrings[0], substrings[1], nil
 }
 
-// addOCIWindowsDevices parses the devices field on the spec and creates corresponding
-// `WindowsDevice` on the container config if valid.
-func addOCIWindowsDevices(g *generator, devs []*runtime.Device) error {
-	for _, device := range devs {
-		prefix, deviceIdentifier, err := getWindowsDeviceInfo(device.HostPath)
-		if err != nil {
-			return err
-		}
-		device := runtimespec.WindowsDevice{
-			ID:     deviceIdentifier,
-			IDType: prefix,
+// hasWindowsDevicePrefix returns true if the device's hostPath contains a prefix.
+// A prefix is any string before "://" in the spec device's `HostPath`.
+func hasWindowsDevicePrefix(hostPath string) bool {
+	substrings := strings.SplitN(hostPath, "://", 2)
```

Comment:
This seems like it could just be `return strings.Contains(hostPath, "://")`.

Reply:
Good point